961 Installing and configuring extlink

Javi-er opened the pull request:
Issue #961 

### Description
This PR installs the [External Links](https://www.drupal.org/project/extlink) Drupal module and configures it.

### Steps to test
1- Login to Tugboat.
2- Create external links in the main navigation, footer, CKEditor fields etc.
3- Validate that the ext link icon is displayed.

Javi-er committed:
Installing and configuring extlink

Javi-er committed:
Merge branch 'main' into 961--install-extlink

Javi-er committed:
Merge branch 'main' into 961--install-extlink

Javi-er committed:
Delete modules/mukurtu_core/config/install/core.entity_form_display.multipage_item.multipage_item.default.yml

Javi-er committed:
Update extlink.settings.yml

Javi-er commented:
@michael-wynne-wsu Tugboat is building now.

michael-wynne-wsu commented:
@Javi-er this looks great, just a few questions/notes:

1) The mailto link in the footer doesn't have an icon. But since there is a larger style icon there maybe it's not necessary? Mailto links in CKeditor fields do have the email icon, just asking about the footer.
2) The Creative Commons field (digital heritage items > permissions and rights > creative commons) is missing the icon. All other fields seem good though!
3) The icon is, for me, very hard to see and identify. Increasing it's size and/or changing the colour may help - what do you think about that?

Otherwise, no other changes needed that I can see.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr978-wjkqrcuerjjdzhl8d27khkmptlalkcfs.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68717638bd4e378c6221fac3

Javi-er committed:
Merge branch 'main' into 961--install-extlink

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr978-wjkqrcuerjjdzhl8d27khkmptlalkcfs.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/68717638bd4e378c6221fac3

Javi-er committed:
Merge branch 'main' into 961--install-extlink

Javi-er committed:
adjusting ext links

Javi-er commented:
@michael-wynne-wsu I made some adjustments, specifically:

1. This is fine, Extlink module will ignore links that contains SVG's in them, so we'll have to handle each case individually, in this case no changes are needed since the link already has the envelop icon in it.
2. This is related to the above, I fixed it so it shows the extlink icon.
3. I made it bigger and changed the color to match the rest of the link color, since it didn't had enough contrast before.

Let me know any other changes.

michael-wynne-wsu reviewed (approved):
@Javi-er perfect, that addresses all of my needs, please merge whenever you're ready!
